activerecord/lib/active_record/associations.rb

```diff
@@ -1250,6 +1250,9 @@ module ClassMethods
         # [collection.create!(attributes = {})]
         #   Does the same as <tt>collection.create</tt>, but raises ActiveRecord::RecordInvalid
         #   if the record is invalid.
+        # [collection.reload]
+        #   Returns an array of all of the associated objects, forcing a database read.
```

Comment:
Hi @sgrif, thanks for the feedback!

I based this on this documentation above https://github.com/yukideluxe/rails/blob/master/activerecord/lib/active_record/associations.rb#L1194-L1196

Both `collection` and `collection.reload` return an `ActiveRecord::Associations::CollectionProxy` so they behave the same. Does it mean that the documentation for `#collection` is also wrong? I'm not sure how to rephrase it, though 🙈